### ISSUE TITLE
[Question] Can custom license functionality be handled using videojs-contrib-dash?

### ISSUE THREAD

**Issue by kirandcunha** [videojs/videojs-contrib-dash#370]
## Description
Briefly describe the issue.
Use a custom license functionality instead of passing just the license URL. 
I have been using inbuilt vhs of videojs 7.0 along with videojs-contrib-eme to successfully play a protected content with a custom license mechanism. videojs-contrib-eme provides getLicense method which helps us. I am trying to migrate to videojs-contrib-dash because of TTML captions support. After adding videojs-contrib-dash plugin, getLicense function is not getting invoked and there is an error
[224][Stream] DRM: No license server URL specified! 

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.

1. Add videojs-contrib-dash and videojs-contrib-eme plugins.
2. Play content which required a custom flow for license acquisition.

### Expected
Please describe what you expected to see.
getLicense function does not get invoked.

### Actual
Please describe what actually happened.
getLicense function should get invoked and license should be fetched from here.

### Error output
If there are any errors at all, please include them here.
[224][Stream] DRM: No license server URL specified! 

## Additional Information
Please include any additional information necessary here. Including the following:

#### videojs
what version of videojs does this occur with?
7.14.3

#### browsers
what browser are affected?
Chrome

#### OSes
what platforms (operating systems and devices) are affected?
Windows

### plugins
are any videojs plugins being used on the page? If so, please list them below.
videojs-contrib-eme

**Comment by gkatsev** [comment]
I don't think that this project works with contrib-eme. I believe everything is handled directly by DASH.js. I'm pretty sure that DASH.js has similar functionality, and if it does you'll probably need to configure it appropriately, see https://github.com/videojs/videojs-contrib-dash#initialization-hook.

**Comment by kirandcunha** [comment]
@gkatsev Thank you. Will check it out.

**Comment by AkashYadav00** [comment]
Hi were you able to fix this?

**Comment by frontend-projects** [comment]
Hi @AkashYadav00 . Did you find any solution to this issue or might have used any hack. I am facing the same issue. Thanks in advance. @gkatsev Can you help to resolve this issue?
